src/views/SearchRedesign/Filter.tsx

```diff
@@ -23,9 +40,34 @@ export const Filter: FunctionComponent<FilterProps> = ({
 
   return (
     <Stack spacing={2}>
-      <Heading as="h4" size="sm">
-        {name}
-      </Heading>
+      <Flex align="center" justify="space-between">
+        <Heading as="h4" size="sm" w="max-content">
+          {name}
+        </Heading>
+        {hint ? (
+          <Popover placement="top-end">
+            <PopoverTrigger>
+              <IconButton
+                aria-label={`Hint: ${name}`}
+                icon={<QuestionIcon h={4} w={4} />}
+                ml={2}
+                px={0}
+                py={0}
+                size="sm"
+                variant="ghost"
+              />
+            </PopoverTrigger>
+            <PopoverContent>
+              <PopoverArrow />
+              <PopoverCloseButton />
+              <PopoverHeader>Hint: {name}</PopoverHeader>
+              <PopoverBody>
+                <Text>{hint}</Text>
+              </PopoverBody>
+            </PopoverContent>
+          </Popover>
+        ) : null}
```

Comment:
I wonder if we should use the same tooltip component/UI across different parts of the website? I noticed they look a little different between screenshots in different PRs. Also the tooltip text here is a larger font size than the text on the sidebar - maybe we could try making the text smaller?